bitsv/network/services/bchsvexplorer.py

```diff
@@ -0,0 +1,83 @@
+import requests
+import json
+from decimal import Decimal
+
+from bitsv.network import currency_to_satoshi
+from bitsv.network.meta import Unspent
+from bitsv.network.transaction import Transaction, TxPart
+
+DEFAULT_TIMEOUT = 30
+BSV_TO_SAT_MULTIPLIER = 100000000
+
+
+class BchSVExplorerDotComAPI:
+    """
+    Simple bitcoin SV REST API --> uses Legacy address format
```

Comment:
Ok, "Legacy address format" is unclear given the context of BSV. Does that mean base58 or cashaddr?

Reply:
It means base58 - I can put it as Legacy address format (base58) to remove ambiguity.

Reply:
Sounds good!